.travis.yml

```diff
@@ -58,3 +58,5 @@ addons:
 #      - ubuntu-toolchain-r-test
     packages:
 #      - g++-4.9
+       # preinstalled for most linux distros, but no in travis
+       - realpath
```

Comment:
Can you move this to install.sh (see line https://github.com/StanfordAHA/CGRAFlow/blob/master/install.sh#L9) for now? That's where all the dependencies are listed, this may change in the future, but for now let's keep in them in one place so we don't lose track of them